src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  font-size: 12px;
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  color: $nav-links-color;
+  text-decoration: none;
+  height: 60px;
+}
+
+.is-active {
+  position: relative;
+  color: $page-design-color;
+}
+
+.is-active::after {
+  position: absolute;
+  content: ' ';
+  width: 100%;
+  height: 4px;
+  background-color: $page-design-color;
+  border-radius: 8px;
+  bottom: 0;
+}
+
+.nav__link:hover {
+  color: $page-design-color;
```

Comment:
The variable `$page-design-color` is used here, but ensure that it is defined elsewhere in your SCSS files. If not, this will cause a compilation error.